components/channel_header/components/header_icon_wrapper.js

```diff
@@ -27,19 +27,19 @@ export default function HeaderIconWrapper({
             class: 'text-nowrap',
             id: 'flaggedTooltip',
             messageID: t('channel_header.flagged'),
-            message: 'See flagged posts',
+            message: 'Flagged posts',
```

Comment:
This and related tooltips were reverted based on https://github.com/mattermost/mattermost-webapp/pull/4830/files#r378063692.  Please check again.